prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -295,6 +298,7 @@ fun main(args: Array<String>) = RouterCli()
         ConvertValuesetsYamlToCSV(),
         ProcessHl7Commands(),
         ValidateTranslationSchemaCommand(),
-        SyncTranslationSchemaCommand()
-    )
+        SyncTranslationSchemaCommand(),
+        ValidateYAMLCommand()
+    ).context { terminal = Terminal(ansiLevel = AnsiLevel.TRUECOLOR) }
```

Comment:
Can now use color support for our output! See examples of `green` and `red` calls in this PR.

<img width="741" alt="Screenshot 2024-05-16 at 4 18 07 PM" src="https://github.com/CDCgov/prime-reportstream/assets/1209453/58cebd5f-2058-47ab-b9e2-81898016cf12">